internal/backend/local/test.go

```diff
@@ -37,6 +37,7 @@ type TestSuiteRunner struct {
 	Config *configs.Config
 
 	GlobalVariables map[string]backend.UnparsedVariableValue
+	GlobalTestVariables map[string]backend.UnparsedVariableValue
```

Comment:
could you add a comment here, clarifying the difference between GlobalVariables and GlobalTestVariables (eg. just something like GlobalVariables comes from the main configuration directory, while GlobalTestVariables are loaded from the test directory).